src/phasorpy/phasor.py

```diff
@@ -34,23 +35,215 @@
 from __future__ import annotations
 
 __all__ = [
-    "phasor_calibrate",
-    "polar_from_reference_phasor",
-    "polar_from_reference",
-    "phasor_to_polar",
-    "phasor_from_lifetime",
-    "phasor_center",
+    'phasor_calibrate',
+    'phasor_center',
+    # 'phasor_from_apparent_lifetime',
+    'phasor_from_lifetime',
+    'phasor_from_polar',
+    'phasor_from_signal_f1',
+    'phasor_semicircle',
+    # 'phasor_to_apparent_lifetime',
+    'phasor_to_polar',
+    'polar_from_reference',
+    'polar_from_reference_phasor',
 ]
 
+import math
+import os
 from typing import TYPE_CHECKING
 
 if TYPE_CHECKING:
-    from ._typing import Any, NDArray, ArrayLike, Literal
-
-import math
+    from ._typing import Any, NDArray, ArrayLike, DTypeLike, Literal
 
 import numpy
 
+from ._phasor import _phasor_from_lifetime, _phasor_from_signal
+
+
+def phasor_from_signal_f1(
+    signal: ArrayLike,
+    /,
+    *,
+    axis: int = 0,
```

Comment:
Just like to mention that this default does not work in many cases, like histograms returned from PTU files. Should we change it to `-1`, to match the default in numpy FFT? `-1` will also b the fastest to compute. On the other hand, `-1` will not work by default with many spectral signals. Thoughts?

Reply:
I thought about that when testing with real data, because I always need to perform some sort of transpose or indicate the axis after reading the files. I think it could be a good idea to leave this default axis as 0, and then maybe modify the read functions so that the returned data contains the information to perform the fft in the first dimension. I believe the spectral data like lsm returns in the first dimension and thats why we started with that workflow. But maybe we can discuss this in the next meeting if it is viable to unify this with spectral and flim.

Reply:
Transposing the signal or modifying the read function to match `axis=0` is a bad idea. Ideally, for huge performance reasons, the `axis` should be contiguous, and that's why numpy defaults to `axis=-1` not only in the fft function. That's a really strong argument for using `axis=-1` as default. But if almost all of our use cases are using `axis=0`, that's a sensible default too.

In principle, `axis` can most often be determined from the xarray coordinates returned by the phasorpy.io read functions. I did not integrate it into this PR since it complicates things. Now that we have implemented the necessary functions, we were also discussing adding higher level functions to `phasorpy.io`, which directly return phasor coordinates from files, like `phasor_from_ptu`. Choosing the axis would be hidden from the user altogether.

Reply:
You are right, those higher level functions may help mitigate the issue I raised so I agree that default to -1 can be a good idea for the axis parameter.

Reply:
I changed the default to `-1` for now. We can discuss further in #20.